src/main/java/at/ac/tuwien/kr/alpha/solver/heuristics/VSIDSWithPhaseSaving.java

```diff
@@ -0,0 +1,226 @@
+/**
+ * Copyright (c) 2018-2019 Siemens AG
```

Comment:
In my opinion, this implementation is independent from `VSIDS` enough for the copyright attribution to Siemens not to be necessary.